### PR TITLE
cleaned up parameters

### PR DESCRIPTION
**Pull request by KevinMenden**
* added missing parameters ("unexpected") to `nextflow.config` and `nextflow_schema.json`
* removed the logging about `single_end` as that's the only supported configuration anway

### CODE REVIEW

**Review by ewels — APPROVED** [review]
Great!

**Comment by ewels** [comment]
Something funky going on with the schema here..

**Comment by KevinMenden** [comment]
Yes already fixed it, just had to remove the default value for `bt_indices`
Otherwise the validation of default parameters gets angry

**Comment by KevinMenden** [comment]
I'll update a few more versions before merging

**Comment by KevinMenden** [comment]
On second thought those are mostly r package which I don't want to touch ... let's leave it at this for now

File: nextflow_schema.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
                 },
                 "bt_indices": {
                     "type": "array",
-                    "default": "null"
```
Comment (ewels) [inline nextflow_schema.json]: @KevinMenden did the `nf-core schema build` command put this here?

Need to fix that behaviour if so..

Comment (ewels) [reply nextflow_schema.json]: https://github.com/nf-core/tools/issues/1074

**Pull request merged**